components/segment-bridge/README.md

```diff
@@ -0,0 +1 @@
+# SEGMENT-BRIDGE
```

Comment:
At least refer to our repo so people know what his is, you can also write a few words...

Reply:
fixed